packages/core-commands/src/admin-navigation-commands.js

```diff
@@ -131,6 +131,37 @@ const getAdminBasicNavigationCommands = () =>
 		};
 	};
 
+const getDashboardCommand = () =>
+	function useDashboardCommand() {
+		const isSiteEditor = getPath( window.location.href )?.includes(
+			'site-editor.php'
+		);
+		const isPostEditor =
+			getPath( window.location.href )?.includes( 'post.php' ) ||
+			getPath( window.location.href )?.includes( 'post-new.php' );
```

Comment:
```suggestion
		const isEditorScreen =
			currentPath?.includes( 'site-editor.php' ) ||
			currentPath?.includes( 'post.php' ) ||
			currentPath?.includes( 'post-new.php' ) ||
			currentPath?.includes( 'customize.php' ) ||
			currentPath?.includes( 'widgets.php' );
```

Can we check other screens that may have the block editor?

Reply:
Sure, I have made the changes.